sdk/go/test/test.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	ehsm "github.com/whh39/ehsm/go"
+	"fmt"
+)
+
+func main() {
+	// export EHSM_APPID=49b84175-1a82-419d-be35-fba6bc85ee69
+	// export EHSM_APIKEY=B8FM3GVex6FAP45yeDuwSvuShjtn3gEJ
+	// export EHSM_ADDR=https://10.112.241.110:9002
+	a := ehsm.NewClient()
```

Comment:
can return an error for the case appid, key, addr are not set?

Reply:
ok, err return will be completed